src/lib/serviceRoutinesV2/postEntities.cpp

```diff
@@ -108,6 +108,14 @@ std::string postEntities
   if (rhttpcode == SccOk || rhttpcode == SccNone)
```

Comment:
Suggested CNR regarding this PR:

"Add: ?type URL parameter in Location header upon entity creation in NGSIv2 (Issue #1765)"